skimage/feature/scattering.py

```diff
@@ -0,0 +1,286 @@
+""" Scattering transform code licensed under
+    BSD license.
+    Original author: Sira Ferradans, based on a code developed by
+    Ivan Dokmanic and Michael Eickenberg
+"""
+import numpy as np
+from skimage._shared.utils import assert_nD
+from skimage.filters.filter_bank import multiresolution_filter_bank_morlet2d
+import warnings
+
+
+def _apply_fourier_mult(signals, filters):
+    """Spatial subsampling on the last two dimensions of X at a
+    rate that depends on 2**j.
+        Parameters
+        ----------
+        signals: ndarray (3D)
+            Signals in the Fourier domain stacked as (num_signals, N,N).
+            Note that color images can be stacked as 'num_signals'.
+        filters: ndarray (3D)
+            Filters in the Fourier domain, stacked as (L,N,N) where
+            L is the num. of filters to apply
+        Returns
+        -------
+        filtered_signals : ndarray
+            Filtered signals in the Fourier domain,
+            stacked as (num_signals,L,N,N)
+        """
+    filtered_s = signals[:, np.newaxis, :, :]*filters[np.newaxis, :, :, :]
```

Comment:
nop, corrected the description of the function. It is just a pointwise multiplication of signals and filters in the Fourier domain